docs/config-options-listings.mdx

```diff
@@ -0,0 +1,38 @@
+---
+id: config-options-listings
+title: Listings For All Buildbuddy Options
+sidebar_label: Options Listings
+---
+
+import FreeServerConfig from '../bazel-bin/server/cmd/buildbuddy/yaml_doc/buildbuddy_server_documented_defaults.mdx'
+import EnterpriseServerConfig from '../bazel-bin/enterprise/server/cmd/server/yaml_doc/buildbuddy_enterprise_server_documented_defaults.mdx'
+import ExecutorConfig from '../bazel-bin/enterprise/server/cmd/executor/yaml_doc/buildbuddy_executor_documented_defaults.mdx'
+
+Provided below are working, documented YAML configs for each Buildbuddy binary
+containing every option that that binary accepts, each set to the default value
+for that option. Any option can be specified on the command line by joining the
+names of the nesting structures, if any, and the option name with a `.`
+separating each element.
+
+E.g.,
+```yaml
+storage:
+    disk:
+        root_directory: /tmp/buildbuddy
+```
+becomes:
+```sh
+buildbuddy -storage.disk.root_directory="/tmp/buildbuddy"
+```
+
+### Buildbuddy Server (Free Tier)
+
+<FreeServerConfig />
```

Comment:
not specific to this PR, but some thoughts on readability of the generated options

```yaml
    tree_cache_seed: treecache # If set, hash this with digests before caching / reading from tree cache type: string
```

this is a bit hard to read because it requires horizontal scrolling, and also it's a bit hard to parse the "type" part because it doesn't end with a period.

what do you think about adding some primitive line-wrapping and also putting the type in parenthesis to more clearly separate it from the help content?

```yaml
    # The desired log level. Logs with a level >= this level will be emitted.
    # One of {'fatal', 'error', 'warn', 'info', 'debug'} (type: string)
    log_level: info
```

Reply:
(or maybe for convenience, you could also list out the full YAML path like:)

```yaml
  # cache.tree_cache_seed (string): If set, hash this with digests before
  # caching / reading from tree cache
  tree_cache_seed: treecache
```

Reply:
sgtm!